ballet/validation/gfssf.py

```diff
@@ -6,22 +6,22 @@
 LAMBDA_2_ADJUSTMENT = 32
 
 
-def _concat_datasets(dfs_by_src, n_samples, omit=None):
+def _concat_datasets(dfs_by_src, n_samples=0, omit=[]):
```

Comment:
never want `omit=[]` in parameter list; instead, `omit=None` and in first line of body, add `if omit is None: omit = []`

See also https://docs.python-guide.org/writing/gotchas/

Reply:
thanks!